### PR TITLE
golang-120: adding package

### PR DESCRIPTION
**Pull request by Toasterson**
I need to find a better method to handle the download of the bootstrap golang from the illumos CI.

The current openindiana golang prodices an error when trying to compile rhis one.

### CODE REVIEW

**Comment by Toasterson** [comment]
Added a shellscript to make things work more smoothly.

PR ready for review.

**Comment by AndWac** [comment]
You'll need to remove golang.p5m from golang-119.

File: components/developer/golang-120/pkg5 [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "dependencies": [
+        "developer/golang-119",
+        "system/library"
+    ],
+    "fmris": [
+        "developer/golang",
+        "developer/golang-119"
+    ],
+    "name": "golang"
+}
```
Comment (AndWac) [inline components/developer/golang-120/pkg5:11]: Wrong version! You'll need to fix the manifest.

File: components/developer/golang-120/golang-120.p5m [hunk 2]
```diff
@@ -0,0 +1,11495 @@
...
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2023 Till Wegmueller
+#
+
+set name=pkg.fmri \
+    value=pkg:/$(COMPONENT_FMRI)-119@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
```
Comment (AndWac) [inline components/developer/golang-120/golang-120.p5m:18]: You'll need to fix the fmri here. Probably better move the complete definition to the Makefile.

File: components/developer/golang-120/pkg5 (hunk 1 above)
Comment (Toasterson) [reply components/developer/golang-120/pkg5:11]: Oh, thanks solved

File: components/developer/golang-120/golang-120.p5m (hunk 2 above)
Comment (Toasterson) [reply components/developer/golang-120/golang-120.p5m:18]: Fixed

**Pull request merged**